backend/backend/management/commands/populate_db.py

```diff
@@ -18,99 +20,85 @@
 
 class Options(TypedDict):
     users: int
-    orgs: int
-    groups: int
-    events: int
+    orgs_per_user: int
+    groups_per_org: int
+    events_per_org: int
 
 
 class Command(BaseCommand):
     help = "Populate the database with dummy data"
 
     def add_arguments(self, parser: ArgumentParser) -> None:
         parser.add_argument("--users", type=int, default=10)
-        parser.add_argument("--opu", type=int, default=1)  # orgs per user
-        parser.add_argument("--gpo", type=int, default=1)  # groups per org
-        parser.add_argument("--epo", type=int, default=1)  # events per org
+        parser.add_argument("--orgs-per-user", type=int, default=1)
+        parser.add_argument("--groups-per-org", type=int, default=1)
+        parser.add_argument("--events-per-org", type=int, default=1)
 
     def handle(self, *args: str, **options: Unpack[Options]) -> None:
-        n_users = options.get("users")
-        n_orgs_per_user = options.get("opu")
-        n_groups_per_org = options.get("gpo")
-        n_events_per_org = options.get("epo")
+        num_users = options.get("users")
+        num_orgs_per_user = options.get("orgs_per_user")
+        num_groups_per_org = options.get("groups_per_org")
+        num_events_per_org = options.get("events_per_org")

```

Comment:
The errors with the `pr_ci_backend` workflow at least, appear to be due to the above 4 possibly being either an `int` or `None` (despite the defaults that are specified in the `parser` arguments on lines L32-L35).

Would doing for instance `options["users"]` as opposed to `options.get("users")` still work? Or at least specifying the default here as in `options.get("users", 10)` so that the `.get()` calls don't possibly return `None` by default.

Reply:
The get method return type is `int | None`. Because the method will return `None` instead of throwing a KeyError, if the key does not exist. We can just use square bracket notation `options["users"]` and there errors are gone 😃.